### ISSUE TITLE
Static drawer disappears at smaller viewports

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly#3780]
At smaller viewports, there's a difference in the behavior between static drawers and not-static drawers.

Not-static drawers will become full width, and the user still has the ability to show/hide the drawer.

Static drawers seem to just disappear. This behavior is visible in the current example: https://www.patternfly.org/v4/components/drawer#static-drawer

Note that static drawers with `position="bottom"` do not disappear. But if a Close button is included in the drawer (as in the current example), this button displays at smaller viewports.

For context, we have a use case where we'd like to use a static inline drawer in a modal, to show additional details about the user options presented in the form in this modal. The expectation is that these details are always visible, at least at larger viewports. At smaller viewports, I'm not sure I know yet what the preferred behavior is :-) between showing the drawer on demand (similar to a not-static drawer), or just stacking the main contents and drawer contents. In my use case, with the drawer being in the context of a modal, it's a bit weird to have a thing with a X button inside of another thing with a X button, so maybe I could just switch to the bottom variant at that viewport to stack the information. 🤷 Any guidance on this specific use case is welcome.

cc @mcarrano @christiemolloy @lboehling

**Comment by mcarrano** [comment]
@jgiardino This is a tough one because the side-by-side drawer is only really useful for wider viewports. But we should have a better answer for how this is intended to respond. Let's start by understanding what is happening now. @mcoker can you take a look at this? Why does the drawer disappear when the viewport shrinks? Was that intentional?

**Comment by mcoker** [comment]
@mcarrano I don't recall, we added that variation with the [primary-detail demo](https://github.com/patternfly/patternfly/pull/2645/) - @mattnolting do you remember what the intent was there?

**Comment by mattnolting** [comment]
TLDR: the example and demos need to be updated. 

@jgiardino @mcarrano @mcoker All drawers are supposed to behave the exact same way at mobile viewports. For the [static drawer example](https://www.patternfly.org/v4/components/drawer#static-drawer), in our documentation only, there needs to be a button (like all other examples) that we manually hide at `md` breakpoint, because there is no dataset to provide that functionality. 

In the context of a demo (`list`, `table`, etc), a dataset exists, and an `item`, which would never be hidden, would trigger the `panel open` action. 

The close button in a `static` or `inline` `drawer panel` hides itself automatically at the `md` breakpoint, unless otherwise prescribed.

I captured the [issue here](https://github.com/patternfly/patternfly-react/issues/4807) and @DaoDaoNoCode is working on [the update](https://github.com/patternfly/patternfly-react/pull/5311).
